pallene/ir.lua

```diff
@@ -177,7 +177,7 @@ local ir_cmd_constructors = {
 
     If      = {"loc", "condition", "then_", "else_"},
     ForNum  = {"loc", "typ", "loop_var", "start", "limit", "step", "body"},
-    ForIn   = {"loc", "typ", "loop_var", "loop_var", "src_tab", "body"},
+    ForIn   = {"loc", "typ", "loop_var", "typ", "loop_var", "src", "body"},--change second loop_var
```

Comment:
We might want to use an array of loop_vars here instead of multiple loop_var variables. By the way, I also just noticed that we might not need the `typ` field, because we can get the type information from the loop_var. Please see PR #183 for an example of how to do that.

Reply:
How to do it? Do I need to just add ```loopvars``` instead of two loop_var?